content/en/notebooks/advanced_analysis/_index.md

```diff
@@ -0,0 +1,122 @@
+
+You can add various cell types to enhance your analysis capabilities. These cells allow you to include additional data sources such as reference tables, use DDSQL to join data, and transform, correlate, and visualize your data effectively. One of the key benefits of this approach is that cells that depend on other cells are automatically updated whenever a dependency changes, ensuring your analysis always reflects the most current data.
+
+### Transformation cell
+
+Add a transformation cell to filter, group, or join, or extract data defined in a data source cell.
+
+1. Type `/transformation` and press <kbd>Enter</kbd> or click on the transform dataset tile at the bottom of the page.   
+2. Select the data source you want to transform in the source dataset dropdown menu.
+
+After adding the transformation cell, you can add any number of transformation operations inside the cell. Choose an operation from the list of supported transformations:
+| Operation | Description |
+|-----------|-------------|
+| Parse | Enter [grok syntax][2] to extract data into a separate column. In the "from" dropdown menu, select the column the data is getting extracted from. |
+| Group | Select what you want to group the data by in the dropdown menus. |
+| Join | Select the type of join, the dataset to join against, and the fields to join on. |
+| Filter | Add a filter query for the dataset. |
+| Calculate | Add a name for the field and the function formula, using the [calculated field expression language][3]. |
+| Limit | Enter the number of rows of the dataset you want to display. |
+| Sort | Select the sort order and column to sort on. |
+| Convert | Select the column and the column type to be converted. |
+
+### Analysis cell
+
+You can also transform your data using SQL by adding an analysis cell to your notebook.
+
+1. Type `/sql` or `/analysis` and press <kbd>Enter</kbd> or click the **Analyze Dataset** tile at the bottom of the page.   
+2. In the source dataset dropdown, select the data source you want to transform.  
+3. Write your SQL query. For supported SQL syntax, see the [DDSQL Reference][4].  
+4. Click **Run** in the top-right corner of the analysis cell to execute your query.
+
+## Viewing and exporting data
+
+For any computational cell that includes a dataset preview, you can view the full 100-row preview by clicking the **View dataset** button.
+
+You can save the results of any computational cell to a Dashboard by clicking **Save to Dashboard** and selecting an existing dashboard or creating a new one. Although this creates a sync between your notebook cell and the exported dashboard graph, changes to the query in your notebook do not automatically update the dashboard. 
+
+If you update the published cell or any upstream cells, a badge appears in the upper-right corner of the cell indicating **unpublished changes**. After you publish those changes, the updates sync to **all** dashboards where the query is used.
+
```

Comment:
Can we also explain how the timeframe works in the dashboard widget that's created. 
we should say that by default, the dataset will be tied to the global timeframe of the dashboard, but that the user has the ability to set a custom timeframe. 
Either way, it's not tied to the timeframe in the notebook